rise/lib/geojson/types.py

```diff
@@ -0,0 +1,17 @@
+# Copyright 2025 Lincoln Institute of Land Policy
+# SPDX-License-Identifier: MIT
+
+from typing import Literal, NotRequired, TypedDict
+
+
+class GeojsonFeature(TypedDict):
+    type: Literal["Feature"]
+    geometry: dict
+    properties: dict
+    id: int
+
+
+class GeojsonResponseDict(TypedDict):
```

Comment:
Could also name this `GeojsonFeatureCollection`
```suggestion
class GeojsonFeatureCollection(TypedDict):
```